### PR TITLE
terraform-modules: Adds tags input for tf managed s3 buckets

### PR DESCRIPTION
**Pull request by userhas404d**
*Issue #, if available:*

*Description of changes:*

Adds tags to tf managed s3 buckets.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by iakov-aws** [comment]
@sean-nixon can you check this as well?

**Review by sean-nixon — CHANGES_REQUESTED** [review]
@userhas404d Can you please also update the module documentation with [terraform-docs](https://terraform-docs.io/)? There is a configuration file already in each module, so all you need to do is point it at each directory (e.g. `terraform-docs terraform-modules/cur-setup-destination`)

**Review by sean-nixon — APPROVED** [review]
LGTM @iakov-aws

File: terraform-modules/cur-setup-destination/README.md [hunk 1]
```diff
@@ -153,14 +162,13 @@ Default:
 }
 ```
 
-### enable\_split\_cost\_allocation\_data
+### tags
```
[the remark this answers is not in the document]
Comment (userhas404d) [reply terraform-modules/cur-setup-destination/README.md:165]: looks like it was just moved: https://github.com/aws-samples/aws-cudos-framework-deployment/pull/885/files#diff-04390840bcd9334e45df1375f94e9e356d3d091fcccdc5bf88b77c4ddcf0c4d2R113

**Review by iakov-aws — APPROVED** [review]

**Comment by iakov-aws** [comment]
Great contribution! TY

**Pull request merged**